data/osp/GarciaLopez_2015_Alexandre/stemma.gv

```diff
@@ -0,0 +1,17 @@
+digraph {
+    original[label="ω"]
```

Comment:
Shouldn't `original` and `archetype` they have `color="grey"` as well ?

Reply:
Yes